dashboard/components/FeatureFlagsList.tsx

```diff
@@ -7,10 +7,10 @@ import {
   Text,
   VStack,
 } from '@chakra-ui/react'
-import {FeatureFlag} from '@toglion/types'
+import {Flag} from 'types'
 
 export interface FeatureFlagsListProps {
-  flags: FeatureFlag[]
```

Comment:
Ideally we'll define these types up higher and use them throughout the app, but right now we're scoping it local to the project.